eslint.config.js

```diff
@@ -134,7 +134,7 @@ module.exports = [
             "eslint-plugin/prefer-placeholders": "error",
             "eslint-plugin/prefer-replace-text": "error",
             "eslint-plugin/report-message-format": ["error", "[^a-z].*\\.$"],
-            "eslint-plugin/require-meta-docs-description": ["error", { pattern: "^(Enforce|Require|Disallow)" }],
+            "eslint-plugin/require-meta-docs-description": ["error", { pattern: "^(Enforce|Require|Disallow) " }],
```

Comment:
How about this to ensure the description also does not end with a space or period?
```suggestion
            "eslint-plugin/require-meta-docs-description": ["error", { pattern: "^(Enforce|Require|Disallow) .+[^. ]$" ,
```

Reply:
....